### PR TITLE
Use streaming md5 library

### PR DESCRIPTION
**Pull request by imurchie**
Otherwise large files will throw an error.

```
FATAL ERROR: invalid table size Allocation failed - process out of memory
```

### CODE REVIEW

**Comment by imurchie** [comment]
The `md5-file` library [purports to be able to hash a 4Gb file](https://github.com/roryrjb/md5-file#md5-file--). I have successfully hashed a 2Gb apk, which is the largest I could build with `ant`.

File: test/fs-specs.js [hunk 1]
```diff
@@ -73,9 +73,15 @@ describe('fs', function () {
...
-  it('md5', async () => {
-    let existingPath = path.resolve(__dirname, 'fs-specs.js');
-    (await fs.md5(existingPath)).should.have.length(32);
+  describe('md5', () => {
+    it('should calculate hash of correct length', async () => {
+      let existingPath = path.resolve(__dirname, 'fs-specs.js');
+      (await fs.md5(existingPath)).should.have.length(32);
+    });
+    it('should be able to run on huge file', async () => {
+      let existingPath = path.resolve(__dirname, '..', '..', 'test', 'assets', 'enormous.apk');
```
Comment (jlipps) [inline test/fs-specs.js]: how big is enormous.apk? do we really want to add it to git? could we also simply generate a huge file that is a list of 1s or something so we know the md5 beforehand and can test it?

File: test/util-specs.js [hunk 2]
```diff
@@ -115,7 +115,7 @@ describe('util', function () {
     });
   });
 
-  describe.only('localIp', function () {
+  describe('localIp', function () {
```
Comment (jlipps) [inline test/util-specs.js]: eek!

**Comment by jlipps** [comment]
one comment then LGTM

**Pull request merged**